### PR TITLE
Feature/1902 process multiple tracks section intersections in parallel

### PR DESCRIPTION
**Pull request by randy-seng**
OP#1902

### CODE REVIEW

File: OTAnalytics/plugin_ui/cli.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def _run_analysis(
         for ottrk_file in tqdm(ottrk_files, desc="Analyzed files", unit=" files"):
             save_path = self._determine_eventlist_save_path(ottrk_file)
             tracks = self._parse_tracks(ottrk_file)
+            # events = self._intersect.run_parallel(tracks, sections)
```
Comment (briemla) [inline OTAnalytics/plugin_ui/cli.py:143]: Is this line necessary?

File: OTAnalytics/plugin_ui/main_application.py [hunk 2]
```diff
@@ -199,6 +202,8 @@ def _create_intersect(self) -> RunIntersect:
             intersect_implementation=ShapelyIntersectImplementationAdapter(
                 ShapelyIntersector()
             ),
+            # intersect_parallelizer=RayIntersectParallelization(),
```
Comment (briemla) [inline OTAnalytics/plugin_ui/main_application.py:205]: Is this line necessary?

**Review by briemla — APPROVED** [review]
Looks good. Only a few commented lines which could be removed.

File: OTAnalytics/plugin_ui/cli.py (hunk 1 above)
Comment (randy-seng) [reply OTAnalytics/plugin_ui/cli.py]: Yes, it's not necessary. I forgot to delete that one!

**Review by briemla — APPROVED** [review]

**Pull request merged**